src/main/java/CollectionsPracticePackage/Person.java

```diff
@@ -0,0 +1,18 @@
+package CollectionsPracticePackage;
+
+public class Person {
+    String name;
+    int age;
+    String nationality;
+
+    public Person(String name, int age, String nationality) {
+        this.name = name;
+        this.age = age;
+        this.nationality = nationality;
+    }
+
+    public String toString() {
+        return String.format("Name " + name + ", " + "age " + age + ", " + "nationality " + nationality);
```

Comment:
Why do you want to use String.format() ?
You can easily : return "Name " + name + ", " + "age " + age + ", " + "nationality " + nationality;

Reply:
Looks like I have a twisted mind :))
Will update!